fenix/app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -362,6 +367,21 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
         return true
     }
 
+    private fun getClickListenerForManageAccount(): Preference.OnPreferenceClickListener {
+        return Preference.OnPreferenceClickListener {
+            viewLifecycleOwner.lifecycleScope.launch(Main) {
+                context?.let {
+                    var url = accountManager.authenticatedAccount()?.getManageAccountURL("menu")
```

Comment:
```suggestion
                    var url = accountManager.authenticatedAccount()?.getManageAccountURL("menu")
```

"menu" is a bit generic, maybe you want to add something more specific like "account-menu" or reuse one of  the other values in `FenixFxAEntryPoint`?